packages/language-server/test/completion-items-utils.ts

```diff
@@ -23,22 +28,29 @@ export function getTagName(textEdit: TextEdit | undefined): string | undefined {
 }
 
 /** Use ⇶ to mark the cursor position */
-export function getSuggestions(
+export async function getSuggestions(
   xmlSnippet: string,
-  ui5SemanticModel: UI5SemanticModel
-): CompletionItem[] {
+  ui5SemanticModel: UI5SemanticModel,
+  settings?: Settings
+): Promise<CompletionItem[]> {
   const { document, position } = getXmlSnippetDocument(xmlSnippet);
   const uri: TextDocumentIdentifier = { uri: "uri" };
   const textDocPositionParams: TextDocumentPositionParams = {
     textDocument: uri,
     position: position,
   };
+  clearSettings();
```

Comment:
having to `clearSettings();` in our test utils may be a warning flag on how we have implemented the state management in the production code.

Reply:
We can clear the settings for the specific document. The state of the settings is global because it's part of a different request, this is how it's managed.

Reply:
Removed